exercises/en/solution_03_05.py

```diff
@@ -0,0 +1,15 @@
+import altair as alt
+import pandas as pd
+
+
+penguins = pd.read_csv('data/penguins.csv').dropna(subset=['sex'])
+
+culmen_layered_plot = alt.Chart(penguins).mark_bar(opacity=0.5).encode(
```

Comment:
`opacity=0.5` does not match the hint (the hint says 0.4)

Reply:
Changed hint here.